plugins/techdocs/src/reader/transformers/addGitFeedbackLink.ts

```diff
@@ -35,20 +35,20 @@ export const addGitFeedbackLink = (configApi: any): Transformer => {
 
     const sourceURL = new URL(sourceAnchor.href);
     const githubHosts = configApi
-      .getConfigArray('integrations.github')
-      .map((integration: any) => integration.data.host);
+      .getOptionalConfigArray('integrations.github')
+      ?.map((integration: any) => integration.data.host);
```

Comment:
This doesn't always read the value correctly, should be `integration.getString("host")` instead. I know it was that way before, but wanna fix as part of this PR? :D Will get rid of the `any` type too

Reply:
I gave it a go, stealing with pride from other locations. WDYT?